apps/dashboard/src/app/(dashboard)/support/components/create-ticket.action.ts

```diff
@@ -40,8 +40,9 @@ function prepareEmailBody(
   product: string,
```

Comment:
lets change this to only take a single argument of type object instead of multiple arguments